DevIncubator-Autopark/Program.cs

```diff
@@ -1 +1,74 @@
-﻿Console.WriteLine("Hello, World!");
+﻿namespace DevIncubatorAutopark
+{
+    class Program
+    {
+        public static void Main()
+        {
+            //1
+            VehicleType[] vehicleTypes = new VehicleType[]
+            {
+               new VehicleType("Bus", 1.2f),
+               new VehicleType("Car"),
+               new VehicleType("Rink", 1.5f),
+               new VehicleType("Tractor", 1.2f)
+            };
+
+            //2
+            foreach (var vehicleType in vehicleTypes)
+            {
+                vehicleType.Display();
+            }
+
+            //3
+            vehicleTypes.Last().TaxCoefficient = 1.3f;
+
+            //4
+            Console.WriteLine($"Max TaxCoefficient = {GetMaxTaxCoefficient(vehicleTypes)}");
+
+            //5
+            Console.WriteLine($"Max TaxCoefficient = {GetAverageTaxCoefficient(vehicleTypes)}");
```

Comment:
Max tax coefficient != Avg tax coefficient.(always check, if you copy some string)

Reply:
fixed